src/container/srv_target.c

```diff
@@ -1178,7 +1198,7 @@ cont_child_destroy_one(void *vin)
 		ABT_mutex_unlock(cont->sc_mutex);
 
 		/* Give chance to DTX reindex ULT for exit. */
-		if (unlikely(cont->sc_dtx_reindex))
+		while (unlikely(cont->sc_dtx_reindex))
 			ABT_thread_yield();
```

Comment:
@Nasf-Fan , reindex is supposed to be very quick? Is this busy yield loop ok?

Reply:
Reindex may take long time. But when close the container, reindex ULT will exit soon when detects the "stopping" flag.